test/version/__snapshots__/version.test.js.snap.webpack5

```diff
@@ -0,0 +1,272 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`single version flag outputs version with b 1`] = `
+"webpack 5.27.2
+webpack-cli 4.5.0
+webpack-dev-server 3.11.2"
+`;
+
+exports[`single version flag outputs version with build 1`] = `
+"webpack 5.27.2
+webpack-cli 4.5.0
+webpack-dev-server 3.11.2"
+`;
+
+exports[`single version flag outputs version with bundle 1`] = `
+"webpack 5.27.2
+webpack-cli 4.5.0
+webpack-dev-server 3.11.2"
+`;
+
+exports[`single version flag outputs version with info 1`] = `
+"@webpack-cli/info 1.2.2
+webpack 5.27.2
+webpack-cli 4.5.0
+webpack-dev-server 3.11.2"
+`;
+
+exports[`single version flag outputs version with info using command alias 1`] = `
+"@webpack-cli/info 1.2.2
+webpack 5.27.2
+webpack-cli 4.5.0
+webpack-dev-server 3.11.2"
+`;
+
+exports[`single version flag outputs version with info using command syntax 1`] = `
+"@webpack-cli/info 1.2.2
+webpack 5.27.2
+webpack-cli 4.5.0
+webpack-dev-server 3.11.2"
+`;
+
+exports[`single version flag outputs version with info using option alias 1`] = `
+"@webpack-cli/info 1.2.2
+webpack 5.27.2
+webpack-cli 4.5.0
+webpack-dev-server 3.11.2"
+`;
+
+exports[`single version flag outputs version with init 1`] = `
+"@webpack-cli/generators 1.3.1
+webpack 5.27.2
+webpack-cli 4.5.0
+webpack-dev-server 3.11.2"
+`;
+
+exports[`single version flag outputs version with loader 1`] = `
+"@webpack-cli/generators 1.3.1
+webpack 5.27.2
+webpack-cli 4.5.0
+webpack-dev-server 3.11.2"
+`;
+
+exports[`single version flag outputs version with migrate 1`] = `
+"@webpack-cli/migrate 1.1.2
+webpack 5.27.2
+webpack-cli 4.5.0
+webpack-dev-server 3.11.2"
+`;
+
+exports[`single version flag outputs version with plugin 1`] = `
+"@webpack-cli/generators 1.3.1
+webpack 5.27.2
+webpack-cli 4.5.0
+webpack-dev-server 3.11.2"
+`;
+
+exports[`single version flag outputs version with serve 1`] = `
+"@webpack-cli/serve 1.3.0
+webpack 5.27.2
```

Comment:
Can we replace versions with `x.x.x` so that we don't have to update snapshots manually every time?

Reply:
yep lets serialize it better